x-pack/legacy/plugins/reporting/server/routes/jobs.ts

```diff
@@ -147,7 +151,47 @@ export function registerJobInfoRoutes(
       const request = makeRequestFacade(legacyRequest);
       const { docId } = request.params;
 
-      let response = await jobResponseHandler(
+      let response = await downloadResponseHandler(
+        request.pre.management.jobTypes,
```

Comment:
Note: this block of code is pretty much duplicated below. I'm not sure if it's worth adding a helper yet, though.

Reply:
I'm ok with dupes, might be refactoring for np later anways